charts/ccsm-helm/templates/curator-cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.retentionPolicy.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ccsm-curator
+  labels:
+    {{- include "ccsm.labels" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.retentionPolicy.schedule | quote }}
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: 120
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: bobrik/curator:5.7.6
```

Comment:
:clown_face: It's a bit disappointing there is no official Docker image yet, even though their repository has a Dockerfile...
:wrench: I would propose using the bitnami variant, as bobrik seems to be a lone developer publishing his own image, and I would expect bitnami to be more thorough when it comes to updating, building, and potentially applying security patches. See https://hub.docker.com/r/bitnami/elasticsearch-curator/

Reply:
Thanks for the input @npepinpe. Seems there were no update since 2 years. Might be worth to migrate to bitnami, but also not sure how reliable it really is. It seems at least the images are smaller, even if they are debian based :eyes: 

Fyi @Sijoma in regards to https://github.com/camunda-cloud/zeebe-controller-k8s/issues/94

Reply:
In my experience bitnami images are very reliable, but I've never used them in prod. I would still expect them to be more reliable than someone's, but of course I could be wrong. Keep in mind that Bitnami has incentive to maintain the ES images since they offer [a production ready K8s experience](https://kubeprod.io/) which installs ELK.

Reply:
Thanks @npepinpe good to know :+1: